inject/src/main/java/io/micronaut/inject/beans/AbstractInitializableBeanIntrospection.java

```diff
@@ -166,6 +166,15 @@ public Builder<B> builder() {
         }
     }
 
+    /**
+     * Whether an accessible constructor exists.
+     * @return True if a default constructor exists
+     * @since 4.7.11
+     */
+    protected boolean hasConstructor() {
+        return false;
```

Comment:
Would be nice to have some default that is compatible with previously compiled beans